vars/integrationArtifactUnDeploy.groovy

```diff
@@ -0,0 +1,11 @@
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/integrationArtifactUnDeploy.yaml'
+
+void call(Map parameters = [:]) {
+    List credentials = [
+        [type: 'token', id: 'cpiApiServiceKeyCredentialsId', env: ['PIPER_apiServiceKey']]
+    ]
+    piperExecuteBin(parameters, STEP_NAME, METADATA_FILE, credentials)
```

Comment:
is there a docker image to run the step?

Reply:
GitHub project for building custom docker image: https://github.com/mayurmohan/IntegrationArtifactUndeploy

Reply:
need to add `parameters = DownloadCacheUtils.injectDownloadCacheInParameters(script, …)` to run on docker.

Reply:
AFAIK all integrationArtifact steps only execute http requests and don't need a docker image to run in. Could you confirm @mayurmohan ?

Reply:
the piper go step should run on an image. the step itself runs the http request.

Reply:
It should run on the jenkins master, no? We did not have to configure this in any integrationArtifact step. Is it really necessary?

Reply:
Yes Oliver,  all integrationArtifact steps only execute http requests.

Reply:
After a brief discussion with @rodibrin we concluded that we don't need any change here.